apps/theme/layouts/forms/DefaultFormLayout/fields/Date.tsx

```diff
@@ -0,0 +1,280 @@
+import React, { useState } from "react";
+import { InputField, StyledInput } from "./Input";
+import { StyledSelect } from "./Select";
+import { Field as FieldWrapper } from "./components/Field";
+import { useBind, BindComponentRenderProp } from "@webiny/form";
+import { FormRenderFbFormModelField } from "@webiny/app-form-builder/types";
+import { FieldLabel } from "./components/FieldLabel";
+import { FieldErrorMessage } from "./components/FieldErrorMessage";
+import styled from "@emotion/styled";
+
+const UTC_TIMEZONES = [
```

Comment:
Noticed this twice, plus also used in `packages/app-headless-cms/src/admin/plugins/fieldRenderers/dateTime/utils.tsx:16`.

Let's move this array into `@webiny/utils` package (`utcTimezones.ts`).